roles/sqs_queue/tasks/destroy.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: List SQS Queues
+  command: aws sqs list-queues --queue-name-prefix "{{ sqs_dead_letter_queue.name }}.fifo"
+  set_fact:
+    queue_to_remove: "{{ command_output.stdout }}"
+
+- name: Destroy fifo SQS queue resources
+  command: aws sqs delete-queue --queue-url "{{ queue_to_remove }}"
```

Comment:
Will the queue destroy with items in it?

Reply:
Yes